statsmodels/graphics/tsaplots.py

```diff
@@ -201,10 +201,7 @@ def seasonal_plot(grouped_x, xticklabels, ylabel=None, ax=None):
     ticks = []
     for season, df in grouped_x:
         df = df.copy() # or sort balks for series. may be better way
-        if pandas.__version__ < '0.17.0':
-            df.sort()
-        else:
-            df.sort_values(inplace=True)
+        df.sort(inplace=True)
```

Comment:
That also doesn't work since TimeSeries sorted inplace by default, so need a change in `compat.pandas`

to be 
```
if version >= '0.17.0':
    def sort_values(df, *args, **kwargs):
        out = df.sort_values(*args, **kwargs)
        return out if out is not None else df
else:
    def sort_values(df, *args, **kwargs):
        out = df.sort(*args, **kwargs)
        return out if out is not None else df
```

I think this should handle all `inplace` combinations.